utils/scripts/ocm/ocm.py

```diff
@@ -450,7 +451,8 @@ def install_addon(self, addon_name="managed-odh",
         """Installs addon"""
         replace_vars = {
                        "CLUSTER_ID": self.cluster_name,
-                       "ADDON_NAME": addon_name
+                       "ADDON_NAME": addon_name,
+                       "NOTIFICATION_EMAIL": self.notification_email 
```

Comment:
could you use the `add_replace_vars` parameter to set the notification email? Because this function should be generic enough to work with any addon. As of now, I don't know if RHOAM would work setting here the notification mail.

You can have a look at `install_rhoam_addon` to see how I used the `add_replace_var` param